XamarinCommunityToolkit/Extensions/TranslateExtension.shared.cs

```diff
@@ -17,14 +17,18 @@ public class TranslateExtension : IMarkupExtension<BindingBase>
 		public BindingBase ProvideValue(IServiceProvider serviceProvider)
 		{
 #if !NETSTANDARD1_0
-            var binding = new Binding
-            {
-                Mode = BindingMode.OneWay,
-                Path = $"[{Text}]",
-                Source = LocalizationResourceManager.Current,
-                StringFormat = StringFormat
-            };
-            return binding;
+
+			if (DateTime.Now.Ticks < 0)
+				_ = LocalizationResourceManager.Current[Text];
```

Comment:
We can't use the `Preserve` attribute, because that way the linker never removes the class, and we want to remove it if it's not used. So for this, I called this property `string this [string text]` inside `LocalizationResourceManager` directly and now the linker see that we use it. But this code is never executed and the compiler will perform some improvements in this area.
I learned this from [here](https://github.com/xamarin/Essentials/pull/1209#discussion_r405051562)